gui/src/components/s3/account-management.vue

```diff
@@ -468,6 +477,122 @@
         </cortx-has-access>
       </v-col>
     </v-row>
+    
+    <v-dialog 
+      v-model="showResetAccountForm" 
+      persistent
+      max-width="500" 
+      id="s3-resetaccount-form">
+      <v-card>
+        <v-card-title class="title mt-6 ml-3">
+          <span>{{ $t("s3.account.reset-password") }}</span>
+          <img id="close-reset-password-dialog"
+            class="cortx-modal-close"
+            :src="require('@/assets/close-green.svg')"
+            @click="closeResetPasswordForm()"
+          />
+        </v-card-title>        
+        <v-divider />
+            <v-col class="col-6 ml-7">
+              <div
+                class="cortx-form-group-custom"
+                :class="{
+                  'cortx-form-group--error':
+                    $v.resetAccountForm.password.$error
+                }"
+              >
+                <label
+                  class="cortx-form-group-label"
+                  for="userPassword"
+                  id="iam-passwordlbl"
+                >
+                  <cortx-info-tooltip
+                    label="Password*"
+                    :message="passwordTooltipMessage"
+                  />
+                </label>
+                <input
+                  class="cortx-form__input_text"
```

Comment:
On some of the the places we are showing water mark text in text box. For example how we show on timezone setting. Lets start using the same way

Reply:
It is pre-existing class and used at 70+ places. We can take other task to update the same later.